Create ui-c8y-1021-30-1-fixed-translation-issue-in-web-sdk-7915.md (#2780)
SoftwareAG/c8y-docs: content/change-logs/application-enablement/ui-c8y-1021-30-1-fixed-translation-issue-in-web-sdk-7915.md

---
date: ""

> title: Fixed translation issue in web sdk (#7915)

Author: The Claude AI suggested to use the following title:
```suggestion
title: Fixed incorrect translations in the web SDK
```

> title: Fixed translation issue in web sdk (#7915)

Reviewer: ```suggestion
title: Fixed issues with custom translations in the Web SDK.
```

> title: Fixed issues with custom translations in the Web SDK.

Reviewer 2: ```suggestion
title: Custom translations for applications now properly displayed 
```

product_area: Application enablement & solutions
change_type:
  - value: change-VSkj2iV9m
    label: Fix
component:
  - value: component-YbYJ3gLU_
    label: Web SDK
build_artifact:
  - value: tc-pjJiURv9Y
    label: ui-c8y
ticket: MTM-62124
version: 1021.30.1
---

> Fixed translation issue in web sdk (#7915)

Author: The Claude AI suggested to use the following body:
```suggestion
The web SDK contained some incorrect translations which could lead to confusing or misleading text being displayed to end users in the UI. This issue has been resolved by updating the translations to the correct values. With this fix, users of applications built using the web SDK will now see the appropriate translated text for their selected language, providing a better localized user experience.
```

> Fixed translation issue in web sdk (#7915)

Reviewer: ```suggestion
Applications using custom ngx translations were not being properly translated. The issue has been resolved by correctly merging translations together. With this fix, users of applications built using the web SDK and using custom ngx translations will now see the appropriate translated text for their selected language.
```

> Applications using custom ngx translations were not being properly translated. The issue has been resolved by correctly merging translations together. With this fix, users of applications built using the web SDK and using custom ngx translations will now see the appropriate translated text for their selected language.

Reviewer 2: ```suggestion
Applications using custom ngx translations were not properly translated. This issue has been resolved. With this fix, applications built using the Web SDK and using custom ngx translations will now properly display the appropriate translated text for the selected language.
```
